packages/nextra/src/client/mdx.ts

```diff
@@ -3,15 +3,19 @@ import {
   useMDXComponents as originalUseMDXComponents
 } from '@mdx-js/react'
 import type { Components } from '@mdx-js/react/lib'
+import { MathJax } from 'better-react-mathjax'
 import Image, { type ImageProps } from 'next/image'
 import { createElement } from 'react'
+import type { ComponentProps } from 'react'
 
 const DEFAULT_COMPONENTS = {
   img: props =>
     createElement(
       typeof props.src === 'object' ? Image : 'img',
       props as ImageProps
-    )
+    ),
+  MathJax: (props: ComponentProps<typeof MathJax>) =>
```

Comment:
I would prefer to add import ast node directly in your rehype plugin, if you don’t know how to do it, I’ll do it for you when I’ll review your PR

Reply:
Is there a place in the code you could point me to where this is done. I looked at the AST for an import statement, and it is fairly long...surely you don't mean for me to hardcode such an AST...?

Reply:
Done